CRM/Utils/Cache/APCcache.php

```diff
@@ -97,10 +101,12 @@ public function set($key, $value, $ttl = NULL) {
    * @return mixed
    */
   public function get($key, $default = NULL) {
-    if ($default !== NULL) {
-      throw new \RuntimeException("FIXME: " . __CLASS__ . "::get() only supports NULL default");
+    CRM_Utils_Cache::assertValidKey($key);
+    $result = apc_fetch($this->_prefix . $key, $success);
```

Comment:
So I'm still under the impression that we will get e-notices sometimes if $success is not predefined - perhaps php versions are different on that?